src/sentry/utils/data_filters.py

```diff
@@ -0,0 +1,48 @@
+"""
+sentry.utils.data_filters.py
+~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2014 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+import fnmatch
+
+
+class FilterTypes(object):
+    ERROR_MESSAGES = 'error_messages'
+    RELEASES = 'releases'
+    BLACKLISTED_IPS = 'blacklisted_ips'
+
+
+def is_valid_release(release, project):
```

Comment:
Fair. My justification for not moving it was the risk of breaking an import somewhere I overlooked, but I suppose tests would ideally cover that. If I'm moving it I might as well update the order of the arguments (and I'm pretty sure it's only called in `coreapi` anyway).